Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -383,12 +383,17 @@ public function findBlockEnd($type, $searchIndex, $findEnd = true)
      * Find tokens of given kind.
      *
      * @param int|array $possibleKind kind or array of kind
+     * @param int|null  $start        optional offset
```

Comment:
idea is to pass `($arg, null, 3)` - and then the dafault value for 2nd parameter is keeped.
sth like `default` keyword proposed for PHp7